src/test/java/picard/util/IntervalListToBedTest.java

```diff
@@ -13,6 +13,7 @@
 public class IntervalListToBedTest {
     private static final String TEST_DATA_DIR = "testdata/picard/util/";
     private final File INTERVAL_LIST = new File(TEST_DATA_DIR, "interval_list_to_bed_test.interval_list");
+    private final File UNSORTED_INTERVAL_LIST = new File(TEST_DATA_DIR, "unsorted_interval_list_to_bed_test.interval_list");
```

Comment:
need to add this file, tests fail currently because doesn't exist in repo

Reply:
🤦‍♀️